UI/data/locale/en-US.ini

```diff
@@ -435,6 +435,8 @@ MissingFiles.Cleared="Cleared"
 MissingFiles.Found="Found"
 MissingFiles.AutoSearch="Additional file matches found"
 MissingFiles.AutoSearchText="OBS has found additional matches for missing files in that directory. Would you like to add them?"
+MissingFiles.NoMissing.Title="No missing files"
```

Comment:
I would suggest `Missing Files Check` as the title, dialog titles are supposed to be Title Case and describe the action being executed, not the result.